optional/vtd/tests/profile_with_vtd-2022.3.py

```diff
@@ -0,0 +1,45 @@
+import os
+from pathlib import Path
+from conans import ConanFile, tools
+
+
+class CloeTest(ConanFile):
+    python_requires = "cloe-launch-profile/[~=0.19.0]@cloe/develop"
+    python_requires_extend = "cloe-launch-profile.Base"
+
+    default_options = {
+        "cloe-engine:server": True,
+    }
+
+    @property
+    def cloe_launch_env(self):
+        return {
+            "CLOE_LOG_LEVEL": "debug",
+            "CLOE_STRICT_MODE": "1",
+            "CLOE_WRITE_OUTPUT": "0",
+        }
+
+    def set_version(self):
+        for version_path in ["../VERSION", "../../../VERSION"]:
+            version_file = Path(self.recipe_folder) / version_path
+            if version_file.exists():
+                self.version = self.project_version(version_path)
+                return
+        git = tools.Git(folder=self.recipe_folder)
+        self.version = git.run("describe --dirty=-dirty")[1:]
+
+    def requirements(self):
+        self.requires(f"cloe-engine/{self.version}@cloe/develop")
+        self.requires(f"cloe-plugin-basic/{self.version}@cloe/develop")
+        self.requires(f"cloe-plugin-noisy-sensor/{self.version}@cloe/develop")
+        self.requires(f"cloe-plugin-speedometer/{self.version}@cloe/develop")
+        self.requires(f"cloe-plugin-virtue/{self.version}@cloe/develop")
+        
+        self.requires(f"cloe-plugin-vtd/{self.version}@cloe/develop")
+        self.requires("vtd-api/2022.3@cloe/stable", override=True)
+        # Runtime requirements for VTD.
+        self.requires("osi-sensor/1.0.0-vtd2.2@cloe/stable")
+        self.requires("vtd/2022.3@cloe-restricted/stable")
+
```

Comment:
```suggestion

# Overrides:
self.requires("zlib/1.2.13", override=True)
```

Reply:
zlib/1.2.13 is a requirement since boost/1.70.0.
Here, we still use boost/1.69.0.
see
https://conan.io/center/boost?version=1.70.0&tab=recipe

Reply:
applied your suggestion